applications/stepswitch/src/stepswitch_bridge.erl

```diff
@@ -319,6 +319,17 @@ build_bridge(#state{endpoints=Endpoints
             ])
           ,wh_json:get_value(<<"Custom-Channel-Vars">>, JObj, wh_json:new())
          ),
+
+    EndpointFilter = fun(Element) ->
+        case Element of
+            {<<"Outbound-Caller-ID-Number">>, Number} -> false;
+            {<<"Outbound-Caller-ID-Name">>, Name}     -> false;
+            _OtherValues                              -> true
+        end
+    end,
+
+    FmtEndpoints = format_endpoints(Endpoints, Number, JObj, AccountId, EndpointFilter),
+
```

Comment:
but this only happens because you are creating the fun here and applying only in line 378 right ?
if you create the name function, then in line 378 you can pass the Name, Number. or am i missing something ?